ruslingdk/db/seeds/howto/skema.html

```diff
@@ -2,19 +2,19 @@
 <ol>
 <li>
 <h5><strong>Log ind p&aring; din Moodle side.</strong></h5>
-<p>Siden er moodle.aau.dk.<br />Brugernavnet er din studiemail&nbsp;<em>(someone17@student.aau.dk),&nbsp;</em>og password&rsquo;et er som default dit CPR-nummer med punktum f&oslash;r og efter&nbsp;<em>(.0102038888.)</em>.</p>
+<p>Siden er moodle.aau.dk.<br />Brugernavnet er din studiemail <em>(someone17@student.aau.dk)</em> og password&rsquo;et er som default dit CPR-nummer med punktum f&oslash;r og efter&nbsp;<em>(.0102038888.)</em>.</p>
```

Comment:
Brug reelle æ ø å her, i stedet for htmlencoded. Vi understøtter det, og det noget pænere og rarere at læse.